common/tools/dev-tool/src/util/browserRelayServer.ts

```diff
@@ -47,7 +53,12 @@ function buildServer(app: Express) {
   app.put("/credential", (req, res) => {
     const id = randomUUID();
     try {
-      const cred = new DefaultAzureCredential(req.body);
+      const cred = new ChainedTokenCredential(
+        new AzurePowerShellCredential(req.body),
+        new AzureCliCredential(req.body),
```

Comment:
az developer CLI credential should also be supported?

Reply:
I'm not entirely sure about the usage scenario for Azure Developer Cli Credential in local development. It definitely is not used in the live test pipeline. @maorleger @KarishmaGhiya @danieljurek Would you have a strong idea on keeping it or not?

Reply:
We don't authenticate azd in the pipeline so it's not necessary for testing purposes in that context. 

I think it might only be useful in local live test contexts. How do you advise users authenticate for local testing?

Reply:
My 2c is that we should err on the side of keeping credentials in that might not be needed, rather than removing credentials we aren't sure about and risking accidentally breaking someone's dev workflow :). If in doubt, let's do what DefaultAzureCredential does, which is to include this one.

Reply:
I was just looking at the credential list to auth via development tools. 
https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#authenticate-via-development-tools

I think we should also add [VisualStudioCodeCredential](https://learn.microsoft.com/javascript/api/@azure/identity/visualstudiocodecredential?view=azure-node-latest) in here.

Though I have not used it, it would be a useful addition. [assuming it works]

Reply:
VisualStudioCodeCredential [does not work](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/TROUBLESHOOTING.md#troubleshoot-visual-studio-code-authentication-issues) and has not worked for like 3 years 😢 

We can skip it, but I do agree with the sentiment of keeping the credential list as close to DAC as posible

Reply:
In that case @maorleger, can you please remove https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#authenticate-via-development-tools ?